package.json

```diff
@@ -228,7 +229,10 @@
 			"dependencies": [
 				"prepare",
 				"./eslint-plugin:build"
-			]
+			],
+			"env": {
+				"NODE_OPTIONS": "--max-old-space-size=8192"
```

Comment:
why do we need this one?

Reply:
The new version of `angular-eslint` or `typescript` was increasing the amount of memory needed to lint the project, this environment variable helps to make sure we do not fail (in local or in the CI).